drivers/boston_reports/app/models/boston_reports/street_to_home.rb

```diff
@@ -361,46 +379,46 @@ def housed_by_cohort
       end
     end
 
-    def matched_by_cohort
-      @matched_by_cohort ||= {}.tap do |data|
-        active_months = stages[matched_string][:scope].pluck(voucher_date_instance.column).
-          reject(&:blank?).
-          map { |d| Date.parse(d).beginning_of_month }.
-          reject { |d| d < Date.new(2010, 1, 1) }. # Ignore voucher dates pre-2010 (they are probably mistakes)
-          uniq.sort
-        months = [active_months.first]
-        month = active_months.first
-        while month < active_months.last
-          month += 1.months
-          months << month
-        end
-        data['type'] = 'line'
-        data['x'] = 'dates'
-        data['columns'] = [['dates'] + months]
-        data['colors'] = {}
-        overall_for_dates = {}
-        cohort_names.each.with_index do |cohort, i|
-          row = [cohort]
-          ids = clients[cohort] & clients[matched_string]
-          dates = report_scope.where(client_id: ids).
-            pluck(voucher_date_instance.column).
-            reject(&:blank?).
-            map { |d| Date.parse(d).beginning_of_month }
-
-          months.each do |month_start|
-            active_count = dates.count { |date| date == month_start }
-            row << active_count
-
-            overall_for_dates[month_start] ||= 0
-            overall_for_dates[month_start] += active_count
-          end
-          data['columns'] << row
-          data['colors'][cohort] = config["breakdown_1_color_#{i}"]
-        end
-        data['columns'] << ['Total'] + overall_for_dates.values
-        data['colors']['Total'] = config['total_color']
-      end
-    end
+    # def matched_by_cohort
```

Comment:
Can this be removed?

Reply:
Yes, but I'm not 100% convinced it won't come back in the next iteration, so leaving for now.